def alg():
    print('POO com python')
    raise Exception('Lancando uma exceção')
    print('Depois da exceção')
    return 'sucesso'